react/utils/client/index.ts

```diff
@@ -53,7 +53,8 @@ export const getState = (runtime: RenderRuntime) => {
 }
 
 export const getClient = (runtime: RenderRuntime, baseURI: string, runtimeContextLink: ApolloLink, ensureSessionLink: ApolloLink, cacheControl?: PageCacheControl) => {
-  const {account, workspace} = runtime
+  const {account, workspace, culture: { locale }, route } = runtime
+  const domain = route.path && route.path.split('/')[1]
```

Comment:
I think it should be `route.id` instead of `route.path`

<img width="747" alt="Screen Shot 2019-09-16 at 15 06 52" src="https://user-images.githubusercontent.com/2726345/64982158-e37e2d00-d893-11e9-93ac-9175f8afcfd3.png">